services/net/transcription/appsettings.json

```diff
@@ -0,0 +1,54 @@
+{
+  "BaseUrl": "/",
+  "Logging": {
+    "LogLevel": {
+      "Default": "Warning",
+      "Microsoft": "Error",
+      "TNO": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "Service": {
+    "MaxFailLimit": 5,
+    "ApiUrl": "http://api:8080",
+    "TimeZone": "UTC",
+    "CapturePath": "/data/capture",
+    "ClipPath": "/data/clip",
+    "AzureRegion": "canadacentral",
+    "AzureCognitiveServicesKey": "a2b1abe07a7c41dd9b3044ec069566d4",
```

Comment:
Don't commit secrets to source code.  You also now need to delete this secret from source code history too

Reply:
This is why I asked you "If I put the azure cognitive services subscription key in .env, how do I access it in C#?" yesterday. I couldn't determine from the code how you insert a value from the .env file into a property in appsettings.json, but I just realized how to do it.